applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_u_pw_small_strain_element.cpp

```diff
@@ -23,11 +23,14 @@
 #include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
 
 #include <boost/numeric/ublas/assignment.hpp>
+#include <custom_utilities/registration_utilities.h>
```

Comment:
Nitpicking:
```suggestion
#include "custom_utilities/registration_utilities.h"
```

In addition, I would separate the system headers from the Kratos headers.

Reply:
Done